static/tuning.js

```diff
@@ -3,27 +3,26 @@
  * @param {string} mode The algorithm for which a paramter is being tuned.
  */
 const tuning = async (mode) => {
-    const url = 'http://127.0.0.1:5000/';
-    const chartIds = ["001","002","004","005"];
-    
-    for (let index = 0; index < chartIds.length; index++){
-        let response = await fetch(url + "tuning/" + mode + "/" 
-            + chartIds[index]);
-        let distances = await response.json();
+  const url = "http://127.0.0.1:5000/";
```

Comment:
This URL shows up everywhere. Let's factor that out into a constant that you can import wherever it's needed. Or better yet, create a fetch wrapper you can use whenever you perform a fetch.